packages/backend/src/config/config.production.ts

```diff
@@ -0,0 +1,29 @@
+import { Env } from '@l2beat/backend-tools'
+
+import { Config } from './Config'
+import { getGitCommitSha } from './getGitCommitSha'
+
+export function getProductionConfig(env: Env): Config {
+  return {
+    name: 'LZMonitoring/Production',
+    logger: {
+      logLevel: 'INFO',
+      format: 'pretty',
```

Comment:
Probably json is the choice here later on

Reply:
Good catch